From 6d37e93dfcb920d25b9c8536f3f8a588566b5c05 Mon Sep 17 00:00:00 2001
From: Mike Christie <michaelc@cs.wisc.edu>
Date: Wed, 29 Jul 2015 04:23:51 -0500
Subject: [PATCH] libceph: fix pr_fmt compile issues
References: fate#318836
Patch-mainline: Not yet, SES2 clustered LIO/RBD

When using ceph from other modules like the target ones, pr_fmt
might already be defined. This just ifndefs it.

Signed-off-by: Mike Christie <michaelc@cs.wisc.edu>
Acked-by: David Disseldorp <ddiss@suse.de>
---
 include/linux/ceph/ceph_debug.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/ceph/ceph_debug.h
+++ b/include/linux/ceph/ceph_debug.h
@@ -1,7 +1,9 @@
 #ifndef _FS_CEPH_DEBUG_H
 #define _FS_CEPH_DEBUG_H
 
+#ifndef pr_fmt
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
+#endif
 
 #include <linux/string.h>
 
